rasa/architecture_prototype/graph_components.py

```diff
@@ -106,63 +115,377 @@ def generate(
 
 
 class StoryToTrainingDataConverter(GraphComponent):
-    def convert_for_training(self, story_graph: StoryGraph) -> TrainingData:
-        messages = []
-        for step in story_graph.story_steps:
-            messages += self._convert_tracker_to_messages(step.events)
-
-        # Workaround: add at least one end to end message to initialize
-        # the `CountVectorizer` for e2e. Alternatives: Store information or simply config
-        messages.append(
-            Message(
-                data=UserUttered(
-                    text="hi", use_text_for_featurization=True
-                ).as_sub_state()
-            )
+    """Provides training data for core's NLU pipeline as well as lookup table buildup.
```

Comment:
Any suggestions? (We mostly call(ed) it "e2e featurization pipeline" but that also seems weird - because tokenizers don't featurize as Featurizers do / we over-use "featurize")